datumaro/plugins/widerface_format.py

```diff
@@ -44,8 +47,13 @@ def __init__(self, path, subset=None):
 
     def _load_categories(self):
         label_cat = LabelCategories()
-        path = osp.join(self._dataset_dir, WiderFacePath.LABELS_FILE)
-        if osp.isfile(path):
+        if has_meta_file(self._dataset_dir):
```

Comment:
Please add any documentation about this feature. It is not obvious that meta files have priority over dataset label map files.

Reply:
Updated documentation.